addons/mrp/models/mrp_workcenter.py

```diff
@@ -34,7 +34,7 @@ class MrpWorkcenter(models.Model):
         'Description')
     default_capacity = fields.Float(
         'Capacity', default=1.0,
-        help="Default number of pieces (in product UoM) that can be produced in parallel (at the same time) at this work center. For example: the capacity is 5 and you need to produce 10 units, then the operation time listed on the BOM will be multiplied by two. However, note that both time before and after production will only be counted once.")
+        help="Default number of pieces (in product unit) that can be produced in parallel (at the same time) at this work center. For example: the capacity is 5 and you need to produce 10 units, then the operation time listed on the BOM will be multiplied by two. However, note that both time before and after production will only be counted once.")
```

Comment:
Given that "unit" is an uom itself, I think we should either stick to "UoM" or "unit of measure"

Reply:
I fear that since the product form view was "improved"  since saas-17.4+, it's not very clear what the product unit is anymore anyways (it's now "inferred" from the sale/cost "units"). So imo we can leave this change as is